Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -227,28 +208,28 @@ open class ChartDataSet: ChartBaseDataSet
     {
         var entries = [ChartDataEntry]()
         
-        var low = 0
-        var high = _values.count - 1
+        var low = values.startIndex
+        var high = values.endIndex - 1
```

Comment:
>When we are able to properly add Collection conformance to ChartDataSet then it will become possible to call this on a subrange of a ChartDataSet where startIndex isn't guaranteed to be 0 and endIndex isn't guaranteed to be count.

Got it. But if a sub set of ChartDataSet is called upon, I think the startIndex and endIndex is still from 0 to count, since we are on a sub set?

Reply:
Depends on how the collection is implemented. If slicing is implemented, then no. `startIndex` and `endIndex` are safer and they work the same as `0` and `count` for standard collection implementations, but still keep it safe for more sophisticated implementations.